rllib/tuned_examples/simple_q/cartpole-simpleq-test.json

```diff
@@ -0,0 +1 @@
+{"cartpole-simpleq-test": {"env": "CartPole-v1", "run": "SimpleQ", "stop": {"episode_reward_mean": 15, "timesteps_total": 500}, "config": {"framework": "tf"}}}
```

Comment:
Should we break this up into >1 lines for better readability?

Reply:
yeah, good point